torch/onnx/symbolic.py

```diff
@@ -555,6 +555,14 @@ def squeeze(g, self, dim=None):
                 dims.append(i)
     else:
         dims = [_get_const(dim, 'i', 'dim')]
+        # Handle negative dims
```

Comment:
actually, before we call type().sizes(), we need to check whether we have the type information of the tensor: something like `if input.type().kind() == "CompleteTensorType" or input.type().kind() == "DimensionedTensorType":`. Check how we did it in softmax case.

Reply:
Thanks! updated with the check